core/embed/rust/src/ui/display/mod.rs

```diff
@@ -803,7 +829,13 @@ pub fn dotted_line(start: Point, width: i16, color: Color) {
     }
 }
 
-pub fn text(baseline: Point, text: &str, font: Font, fg_color: Color, bg_color: Color) {
+/// Paints a pixel with a specific color on a given point.
+pub fn paint_point(point: &Point, color: Color) {
```

Comment:
Naive question, what are the benefits of using `set_window & pixeldata` compared to this? It will probably be quicker, but for smaller things like rectangle corners (the only place where it is used) this one seems more simple.

Reply:
Yes, it's mostly about speed, and that I wouldn't want encourage future people to use `paint_point` as it's the wrong thing to do in most cases. Rectangle corners are fine, I would just maybe use 4x `rect_fill()` directly there, but it's a nitpick.

Reply:
> I wouldn't want encourage future people to use paint_point as it's the wrong thing to do in most cases

hard agree, and we can use `display::bar` in the `rect_fill_corners` directly